synapse/models/infotech.py

```diff
@@ -2582,7 +2590,44 @@ def getModelDefs(self):
                     ('sandbox:file', ('file:bytes', {}), {
                         'doc': 'The initial sample given to a sandbox environment to analyze.'
                     }),
+
+                    # TODO
+                    # ('windows:task', ('it:os:windows:task', {}), {
+                    #     'doc': 'The Microsoft Windows scheduled task responsible for starting the process.'}),
+
+                    ('windows:service', ('it:os:windows:service', {}), {
+                        'doc': 'The Microsoft Windows service responsible for starting the process.'}),
+                )),
+
+                ('it:os:windows:service', {}, (
+
+                    ('host', ('it:host', {}), {
```

Comment:
Is this better treated like a light edge? I could see **one** service configuration being the same across multiple hosts.

Reply:
( since windows services are themselves an entity resolution problem, i'll leave comparison of them being "the same" to analyst logic )